jolt-core/src/jolt/instruction/virtual_assert_eq_signs.rs

```diff
@@ -0,0 +1,88 @@
+use crate::poly::field::JoltField;
+use rand::prelude::StdRng;
+use rand::RngCore;
+use serde::{Deserialize, Serialize};
+
+use super::{JoltInstruction, SubtableIndices};
+use crate::{
+    jolt::subtable::{eq_msb::EqMSBSubtable, LassoSubtable},
+    utils::instruction_utils::chunk_and_concatenate_operands,
+};
+
+#[derive(Copy, Clone, Default, Debug, Serialize, Deserialize)]
+pub struct EQSIGNSInstruction(pub u64, pub u64);
+
```

Comment:
Can we call this `ASSERTEQSIGNSInstruction` to stay consistent with the pattern?